### ISSUE TITLE
[Group] Build Leaflet Map with Features

### ISSUE THREAD

**Issue by cityhubla** [maptimeLA/craftmap_adventure#14]
A leaflet map with features in geojson format loaded here on this repo<

Here is the link to the first draft with Phase 1 overlay
[Leaflet Map](https://maptimela.github.io/craftmap_adventure/map/phase_1/phase_1.html)

![image](https://cloud.githubusercontent.com/assets/6407796/20907614/2fa49508-bb04-11e6-9239-58795b1d66f8.png)

**Comment by cityhubla** [comment]
Scale: Level 13

**Comment by matikin9** [comment]
Feedback:

- Printed perfectly to a 6"x6" square for me.
- Highways, parks, and water are clearly visible through a thick linen.
- Major streets and contour lines are not visible through a thick linen.
- Parks and Water are difficult to distinguish through a thick linen.
- Features may be more distinguishable with the thin cotton muslin, will try that tonight.

**Comment by cityhubla** [comment]
Great! I'll update the map with thicker lines and add the labels

**Comment by JRHutson** [comment]
Is the plan to have a different tile set load when you zoom out? I'm getting nothing when I zoom out, making it difficult to orient myself or find a particular area.

**Comment by matikin9** [comment]
@JRHutson I believe that the zooming doesn't work because it is just a single tileset from the one zoom level that matches the scale at which we want the tiles.

**Comment by cityhubla** [comment]
@JRHutson, yup its only at one zoom level, 13 I believe. I should restrict that in the code.

**Comment by colombene** [comment]
FYI, I'm getting a problem I isolated to Safari with the printed area being too high and
safarimapprint cutting off at the top of the page (see below). It works in Chrome though.

![safarimapprint](https://cloud.githubusercontent.com/assets/10321383/21416072/c1a10000-c7c4-11e6-82bb-3c0937b12285.png)
